app/services/ExerciseService.ts

```diff
@@ -1,6 +1,11 @@
 import { marked } from "marked";
 import NotFoundException from "../exceptions/NotFoundException";
 import fs from "fs";
+import { string } from "zod";
```

Comment:
We most liekly don't want to be importing the data type from our input validation library.

This was probably an automatic import.